htdocs/product/inventory/inventory.php

```diff
@@ -1086,7 +1086,7 @@ function barcodeserialforproduct(tabproduct,index,element,barcodeproductqty,sele
 					print '<td class="right">';
 
 
-					if (! empty($obj->pmp_real)) $pmp_real = $obj->pmp_real;
+					if ($obj->pmp_real >= 0) $pmp_real = $obj->pmp_real;
```

Comment:
This test seems to be always true. When is it false ?

Reply:
You're right, this condition seems to always be true. But, on the inventory lines, the pmp is not displayed when it is equal to 0. However, it seems important to display this information (rather than leaving the information empty)

Reply:
What should I do ? remove the test that seems to be always true ?

Reply:
@eldy

Reply:
> You're right, this condition seems to always be true. But, on the inventory lines, the pmp is not displayed when it is equal to 0. However, it seems important to display this information (rather than leaving the information empty)

Yes when value is 0 in database, it means value was never initialized or defined. So we don't have to show it to keep only non null value so screen is cleaner to show only data that has a value.

Reply:
Here I'm talking about entering the PMP (real PMP) as 0.
When I do this, the test prevents this data from being recorded (which then shows the expected PMP)

Reply:
@eldy

Reply:
@eldy

Reply:
@eldy

Reply:
If goal is to make a difference between PMP entered with value 0 with no PMP entered, then you can test if value is '' or '0'.

Reply:
Done @eldy